### PR TITLE
Ms/update ruby and deps

### PR DESCRIPTION
**Pull request by tmikeschu**
Ahoy! I got an email from heroku about a stack version being deprecated, so I've been going through and updating stuff!

This PR:

* Defines a ruby version (2.6.2)
* Updates gems
* Fixes deprecations

### CODE REVIEW

**Comment by tmikeschu** [comment]
@DavidKnott hey man! Would you be up for indulging me in a review/merge?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -56,53 +46,51 @@
...
-  #     --seed 1234
-  config.order = :random
-
-  # Seed global randomization in this process using the `--seed` CLI option.
-  # Setting this allows you to use `--seed` to deterministically reproduce
-  # test failures related to randomization by passing the same `--seed` value
-  # as the one that triggered the failure.
-  Kernel.srand config.seed
-=end
+  # The settings below are suggested to provide a good initial experience
```
Comment (ski-climb) [inline spec/spec_helper.rb:49]: @tmikeschu Was it intentional to comment out this config file?

**Review by ski-climb — COMMENTED** [review]
Thanks for tackling the tech debt and getting the project up and running again!  Just had one question where it looks like a lot of the config for SimpleCov is commented out - not sure if that was intentional or not.

File: spec/spec_helper.rb (hunk 1 above)
Comment (tmikeschu) [reply spec/spec_helper.rb:49]: Twas! I think it was a tricky diff block because it (standardrb) formatted `=begin =end` to be single line `#` comments.

Comment (tmikeschu) [reply spec/spec_helper.rb:49]: I just saved and standardrb took it from there.

**Review by ski-climb — APPROVED** [review]

**Comment by tmikeschu** [comment]
Thanks @ski-climb !

**Pull request merged**